packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -94,11 +117,15 @@ export const render = async (req, res, next) => {
     const WrappedApp = routeComponent(App, false, res.locals)
 
     const [pathname, search] = req.originalUrl.split('?')
-    const location = {
+    let location = {
```

Comment:
We are changing property inside the location object, const is still valid

Reply:
Artifact of going to different solution iterations. Fixed it.